templates/segs_demo.html

```diff
@@ -88,10 +101,13 @@
 		'json');
 }
 
+
 function sigmoid(t) {
-    return 1/(1+Math.pow(Math.E, -t));
+	// re-scaled illustration
+    return 1/(1+Math.pow(Math.E, -0.1*t));
 }
 
+
 function geteventcb(data){
 	if(data.status != 0){
 		alert(data.data);
```

Comment:
Replace the following code with the function call to the render_roadsegments function. 

Also add information window, with the information illustrated in step2 of scenario 2. 

https://docs.google.com/document/d/1fmPTSg4u0wrEcPr8d2NBQoBAX0HVnGTnp5zt085a3zQ/edit